.github/workflows/benchmarks.yml

```diff
@@ -44,12 +44,36 @@ jobs:
           path: benchmarks/timings_1.pickle
       - name: Checkout PR branch
         uses: actions/checkout@v4
+        if: github.event_name == 'pull_request_target'
         with:
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
           fetch-depth: 0
           persist-credentials: false
           clean: false
+      - name: Checkout PR branch from issue comment
```

Comment:
It's breathtaking that GH Actions is so expressive that it enables you to automate doing this. Reminds me of this [post](https://news.ycombinator.com/item?id=38938470).

Reply:
Yeah it can be insane.